src/test/resources/features/complaints/ComplaintsEndToEnd.feature

```diff
@@ -62,6 +73,17 @@ Feature: Complaints End To End
       | Ex-Gratia        | Response Draft |
       | Minor Misconduct | Response Draft |
 
+  Scenario Outline: User moves a BF case to the Draft stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "<complaintType> Draft" stage
```

Comment:
Same as with the Triage scenario, this looks like a better fit for iCreateACOMPCaseForAComplaintAndMoveItToStage